.github/workflows/docker-push-to-registries.yml

```diff
@@ -70,6 +70,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Get SBOM from artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ github.run_number }}-sbom.json
```

Comment:
Note: On my shared-workflows branch I had to add `secrets: inherit`,[but it still failed](https://github.com/telicent-oss/telicent-instance/actions/runs/9194592999/job/25288742612#step:5:23)

Reply:
Makes sense as far as I can tell there is no `telicent/telicent-instance-build:latest` image

Reply:
Hm

I thought it would be available [here](https://hub.docker.com/r/telicent/telicent-instance-build).  i.e. it seems trivy should do `docker pull telicent/telicent-instance-build`

I retried, [same result](https://github.com/telicent-oss/telicent-instance/actions/runs/9194592999/job/25318568013)

hm.

> as far as I can tell

How did you verify?

Reply:
Could it be that the command is resolve to is `trivy -i` input (file?) rather then `trivy [image]` (which is hooked up to docker tooling)

```
NAME:
   trivy image - scan an image

USAGE:
   trivy image [command options] image_name

OPTIONS:
   --input value, -i value          input file path instead of image name [$TRIVY_INPUT]
```